static/app/views/replays/replayTable.tsx

```diff
@@ -92,7 +94,7 @@ function ReplayTable({replayList}: Props) {
                   <Link
                     to={`/organizations/${organization.slug}/replays/${generateEventSlug({
                       project: replay.project,
-                      id: replay.id,
+                      id: idKey ? replay[idKey] : replay.id,
```

Comment:
let's be explicit + consistent and make `idKey` required (or default to `"id"`)

Reply:
choose to make `idKey` required.